cmd/lnmuxd/run.go

```diff
@@ -48,20 +48,75 @@ func runAction(c *cli.Context) error {
 		return err
 	}
 
-	releaseLock, err := initDistributedLock(&cfg.DistributedLock)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	group, ctx := errgroup.WithContext(ctx)
+
+	err = run(ctx, cfg, group)
+	if err != nil {
+		cancel()
+
+		// We don't need to catch the error since we only want to wait all goroutines to finish
+		group.Wait() // nolint: errcheck
```

Comment:
If you do `_ = group.Wait()`, doesn't the linter error go away?

Reply:
Yes, indeed.

Usually I use // nolint: errcheck instead of `_ = xxxxx` (since we want to avoid linter error), but seeing your comment you prefer the other way!
I will change it

Reply:
Yes, I think generally that is the preferred way. To solve it in code.